app/resource_builders/dlme_json_resource_builder.rb

```diff
@@ -22,17 +23,42 @@ def to_solr
       transform_to_untokenized_solr_fields(resource.metadata, sink: sink)
 
       TOKENIZED_COPY_FIELDS.each do |key|
-        sink["#{key}_tsim"] = source[key] if source[key]
+        next unless source[key]
+
+        if source[key].is_a?(Hash)
+          source[key].each do |language_code, values|
+            sink["#{key}.#{language_code}_tsim"] = values
```

Comment:
This brings symmetry between the lang-aware `_ssim` fields and the lang-aware `_tsim` fields.

Reply:
ok, and where is the test for THIS change?

Reply:
I believe lines 62-64 test this code.